packages/components/src/slot-fill/index.js

```diff
@@ -3,11 +3,11 @@
  */
 import Slot from './slot';
 import Fill from './fill';
-import Provider from './provider';
+import Provider, { Consumer } from './provider';
 
 export { Slot };
 export { Fill };
-export { Provider };
+export { Provider, Consumer };
```

Comment:
I originally used `SlotFillConsumer` instead of `Consumer`. Changed the name as to be more in line with `Provider` naming.